plugin/lspconfig.lua

```diff
@@ -175,8 +175,8 @@ if vim.version.ge(vim.version(), { 0, 11, 2 }) then
       end
     end
   end, {
-    desc = 'Disable and stop the given client(s)',
-    nargs = '*',
+    desc = 'Disable and stop the given client',
+    nargs = '?',
```

Comment:
Oh, I thought it would simplify the implementation too :) 

well, let's try it for now and see how it goes.

Reply:
Yeah we still need to handle multiple servers in the default case with no arguments :grinning: 

Thanks for the merge!